src/test/42-function.ts

```diff
@@ -169,8 +169,12 @@ describe("validating function endpoints", function() {
       try {
         await c8Client.function.removeFunctionWorker("testSdkEvStreamAdhoc");
       } catch (err) {
-        expect(err.code).to.equal(403);
-        expect(err).to.have.property("message", "Forbidden");
+        let message;
+        if (err instanceof Error) {
+          expect(err).to.have.property("code", 403);
+          expect(err).to.have.property("message", "Forbidden");
+        } else message = String(err);
+        console.log(message);
```

Comment:
@dlozina-macrometa message would be undefined and logged if went inside if... the same thing which you had earlier did message=String(err) can be applied here as well?

Reply:
👍